release_notes/ReleaseNotes_Upcoming.md

```diff
@@ -113,7 +113,8 @@ See [here](https://github.com/ISISComputingGroup/IBEX/wiki#instrument-informatio
 | [#8382](https://github.com/ISISComputingGroup/IBEX/issues/8382) | Minor | Fix error message running genie_python.bat caused by ipython update  |
 | [#8372](https://github.com/ISISComputingGroup/IBEX/issues/8372) | Patch | Reduce memory use for long-running scripts using matplotlib's interactive mode (`pyplot.ion()`). In particular, this addresses a memory leak in Muon background plots. |
 | [no ticket](https://github.com/ISISComputingGroup/genie_python/pull/414) | Patch | Allow passing `use_numpy` to `g.get_pv` to return `numpy` arrays for EPICS waveform data. |
-|[#8411](https://github.com/ISISComputingGroup/IBEX/issues/8411) | Minor | Adds support for redefining motor positions via CLI |
+| [#8411](https://github.com/ISISComputingGroup/IBEX/issues/8411) | Minor | Adds support for redefining motor positions via CLI |
+| [#8415](https://github.com/ISISComputingGroup/IBEX/issues/8411) | Minor | Adds pyright to genie script checker |
```

Comment:
As this is a very user-facing change, it might be good to put it under the "highlights" section, with an expanded (user-facing) explanation of what `pyright` does & why they should care about it.

From a scientist perspective, the first interaction they have with this code is likely to be a red "error" message, so we should be careful to explain in these notes what it's checking and why these checks are a good thing.